experimental/traffic-portal/src/app/shared/generic-table/generic-table.component.ts

```diff
@@ -629,7 +660,7 @@ export class GenericTableComponent<T> implements OnInit, OnDestroy {
 	}
 
 	/**
-	 * Builds a link for a link context menu item.
+	 * Builds a link for a link context menu item. Used to prevent page reload when navigating on the same tab.
```

Comment:
it's actually not used for anything because the href it builds is never used because you're preventing default behavior on the click event

Reply:
If the context menu link is set to `newTab`, it does.